Mathlib/LinearAlgebra/Dual.lean

```diff
@@ -340,11 +336,9 @@ theorem toDual_injective : Injective b.toDual := fun x y h ↦ b.ext_elem_iff.mp
 theorem toDual_inj (m : M) (a : b.toDual m = 0) : m = 0 :=
   b.toDual_injective (by rwa [_root_.map_zero])
 
--- Porting note (https://github.com/leanprover-community/mathlib4/issues/11036): broken dot notation https://github.com/leanprover/lean4/issues/1629 LinearMap.ker
 theorem toDual_ker : LinearMap.ker b.toDual = ⊥ :=
```

Comment:
I think dot notation not working if the type cannot be found is still something of a regression from Lean 3. I would keep this porting note (also below).

Also, if Yaël had their way, we'd be turning the `LinearMapClass` argument to `LinearMap.ker` back into a plain linear map, so dot notation would also work then.

Reply:
That is true. I was thinking that if we make that change, then we would need a systematic refactor anyways, and these few porting notes wouldn't be too helpful.

Reply:
Hmm, I suppose there are indeed many more places where we would have to reapply dot notation. But I still prefer those places to be easily findable, and a porting note is one method. As long as there is some useful way to find dotnotationifiable occurrences, I'm not going to block merging this PR over this.

Reply:
Yeah, I think we can find this automatically later if it becomes relevant.